.github/workflows/Spring-develop-CD.yml

```
@@ -0,0 +1,66 @@
+
+      - name: SSH로 EC2에 접속하기
+        uses: appleboy/ssh-action@v0.1.7
+        with:
+          host: ${{ secrets.EC2_HOST }}
+          username: ${{ secrets.EC2_USER }}
+          key: ${{ secrets.EC2_SSH_KEY }}
+          script_stop: true
+          script: |
+            cd clue
+            sudo fuser -k -n tcp 8080 || true
+            sudo nohup java -jar cd.jar > ./output.log 2>&1 &
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)